internal/store/postgres/store_test_util.go

```diff
@@ -1,12 +1,11 @@
-package postgres_test
+package postgres
```

Comment:
can we keep the package to `postgres_test`

Reply:
the postgres test here is being used for repository test which now used on core for report repository. Otherwise, we need to copy some codes here for the test on repository out of this package.